core/src/main/scala/org/locationtech/rasterframes/expressions/transformers/Mask.scala

```diff
@@ -73,6 +73,15 @@ abstract class Mask(val left: Expression, val middle: Expression, val right: Exp
   override protected def nullSafeEval(targetInput: Any, maskInput: Any, maskValueInput: Any): Any = {
     implicit val tileSer = TileUDT.tileSerializer
     val (targetTile, targetCtx) = tileExtractor(targetExp.dataType)(row(targetInput))
+
+    // Which of these is preferable? companion object Seq contains or pattern match on trait?
+//    assert(!CellType.noNoDataCellTypes.contains(targetTile.cellType), maskErrorStr)
+    targetTile.cellType match {
+      case _: NoNoData ⇒ assert(false,
```

Comment:
See comment below about `AssertionError`. The convention I see in Scala is that `require` is more common for user-sourced errors, and `assert` is for trapping coder-sourced errors. So I think `assert` should be changed to `require`.